### ISSUE TITLE
Performance Improvements

### ISSUE THREAD

**Issue by philsturgeon** [stoplightio/stoplight.io#1488]
As we've worked on making our marketing site more awesome, we've made it a bunch slower.

Here are the Google PageSpeed Rankings

- `/` is 28/100: https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fstoplight.io%2F
- `/open-source/spectral` is 20/100: https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fstoplight.io%2Fopen-source%2Fspectral
- `/open-source/prism` is 28/100: https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fstoplight.io%2Fopen-source%2Fprism

A lot of it is really simple stuff, for example the sendgrid logo is 3300 pixels wide, but we're only displaying it as 100px wide, so we could just shrink it (then run through ImageOptim again!). 

There's another logo which isnt showing on the homepage for me at all (logo_appointmentplus_transparent.png) and thats 1000px. 

<img width="783" alt="image" src="https://user-images.githubusercontent.com/67381/85692285-927a4f80-b6d5-11ea-942a-fb789703390b.png">

Really we need to be in the 90's here, for SEO benefits as well as user satisfaction. We can send multiple PRs for multiple fixes to improve performance over time, and treat this issue more like an epic.

Other than that the unused JavaScript is a similar problem that we have on Platform so grab @lottamus or @filipebarcos to share some ideas on what was done there.

**Comment by mnaumanali94** [comment]
Agreed. For starters i’m going to be compressing all images used in the website today. We can do a quick run then.

**Comment by ksatx** [comment]
[GA Site Speed Suggestions Report](https://analytics.google.com/analytics/web/?authuser=1#/savedreport/h4D-5Dk7R1SmZYR9zuLaiA/a73790375w198010246p192731154/_u.date00=20200525&_u.date01=20200625&_r.dsa=1&_.advseg=&_.useg=&_.sectionId=)
